config/initializers/hyrax.rb

```diff
@@ -1,4 +1,5 @@
-# Use this to override any Hyrax configuration from the Knapsack
+MOBIUS_CONCERNS = ['MobiusWork'].freeze
 
 Hyrax.config do |config|
+  config.register_curation_concern MOBIUS_CONCERNS.map { |concern| concern.underscore.to_sym }
```

Comment:
mobius isn't the only tenant

Reply:
The other concerns get registered via Hyku.